ops/k8s/README.md

```diff
@@ -0,0 +1,119 @@
+
+* postgres: A simple postgresql 14 service using a small persistent volume.
+* localstack: A simulator for S3, SSM, and SQS.
+
+The other three are BFD applications that can be deployed for testing:
+
+* migrator: The bfd-db-migrator program running inside kubernetes as a batch job.
+* pipeline: The bfd-pipeline-app program running inside kubernetes as a batch job.
+* server: The bfd-server-launcher/bfd-server-war program running inside kubernetes as a service.
+
+All three of these programs are intended to be deployed using their corresponding run script in `/apps/utils/scripts`.  These scripts have been modified to include a `-k` command line option that directs the script to configure the program using the simulated SSM in the cluster and then deploy the app using resources in this directory.
+
+## Getting Started
+
+For local development you need to have either Rancher Desktop or Docker Desktop installed and kubernetes enabled.  You also must have `kubectl` and `docker` commands installed and in your PATH.  It is possible that other kubernetes distributions will work as well but only Rancher Desktop has been tested so far.
+
+### Special Notes for Rancher Desktop
+
+Be sure to use the versions of `kubectl` and `docker` installed by Rancher Desktop.  These are installed in `$HOME/.rd/bin` on Mac when using `brew install rancher` to install RD.  Also be sure to set your docker context to point to Rancher Desktop rather than the default.  Unset the `DOCKER_HOST` environment variable if you have it set since it overrides the docker context setting.
+
+There are known issues with running testcontainers inside of RD.  You may need to run normal builds in a shell using colima directly and limit your use of the shell using Rancher Desktop to running the applications and interacting with kubernetes.
+
+### Installing the external service apps
+
+In a real cluster the database and AWS services would be external to the cluster.  For local development we use local versions so that we can test without being connected to the internet or using external resources.
+
+All commands in this document assume that your current working directory is `ops/k8s`.
+
+First you need to create the namespace used to run all of the applications.
+
+```sh
+kubectl create namespace dev
+```
+
+Next you need to start the database and AWS services.
+
+```sh
+helm -n dev install postgres helm/postgres
+helm -n dev install localstack helm/localstack
+```
+
+At this point you should see both charts have been installed.
+
+```sh
+$ helm -n dev list
+NAME      	NAMESPACE	REVISION	UPDATED                             	STATUS  	CHART           	APP VERSION
+localstack	default  	1       	2023-10-23 09:12:25.892621 -0400 EDT	deployed	localstack-0.1.0	2.2.0      
+postgres  	default  	1       	2023-10-23 09:12:06.429432 -0400 EDT	deployed	postgres-0.1.0  	14         
+```
+
```

Comment:
On your list already, but reminder to add the bit about building the images before the next step! Maybe also with the -x to avoid that permissions issue on that one script.